python/Ganga/Utility/Shell.py

```diff
@@ -40,6 +40,9 @@
 import time
 import signal
 import subprocess
+
+from Ganga.Utility.execute import execute
```

Comment:
If we're going to import this why not make most of the functions here wrap around to it as well? It would be nice to have just 1 place handling all of the subprocess calls so we don't have to worry about small bugs